view.md

```diff
@@ -0,0 +1,195 @@
+# view
+
+TiDB supports views, A view acts as a virtual table,  It can be created from `SELECT`  statements. On the one hand, Using the view can only expose safe columns and data to users, to ensure the security of sensitive columns and data in the underlying table.
+
+On the other hand, defining complex queries that frequently appear as views can make complex queries simpler and more convenient.
```

Comment:
This appears to be very similar to views.md which also exists in the PR. Please pick one (I prefer using the plural name). Thanks!

Reply:
view.md deleted